frontend/components/panels/GrossProfitCheckPanel.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name" :disabled="!isExplanationsShown">
+        <template #title> {{ currentStateAndTitle.title }} </template>
+        <TextBlock v-if="isExplanationsShown">
+            You cannot execute an auction while it is not yet profitable. Once the auction price drops below the price
+            on UniSwap, you may continue with the bidding process
```

Comment:
NIT: instead of "with the bidding process" I would write "with auction participation". Rationale: Since the grossprofitcheck is only used for the swap transaction flow a user doesn't bid.